cloudbuild/README.MD

```diff
@@ -0,0 +1,21 @@
+# Continuous Integration via Cloud Build
```

Comment:
Can you mention a bit about how we store/encode and use the SSH key?

Reply:
Done.